lua/nvim-lsp-ts-utils/import-all.lua

```diff
@@ -177,30 +177,50 @@ local response_handler_factory = function(callback)
     end
 end
 
+-- if a module has more than one import edit returns true
+local should_reorder = function(edits)
+    local modules = {}
+    for _, edit in ipairs(edits) do
+        if edit.newText then
+            for module in string.gmatch(edit.newText, "import.*from (.+)[;\n]?") do
+                local import = module:gsub("\n", "")
+                if modules[import] then
+                    return true
+                end
+                modules[import] = true
+                break
+            end
+        end
+    end
+    return false
+end
+
 local apply_edits = function(edits, bufnr)
     if vim.tbl_count(edits) == 0 then
         return
     end
 
     lsp.util.apply_text_edits(edits, bufnr)
 
-    -- organize imports to merge separate import statements from the same file
-    require("nvim-lsp-ts-utils.organize-imports").async(bufnr, function()
-        -- remove empty lines created by merge
-        local empty_start, empty_end
-        for i, line in ipairs(api.nvim_buf_get_lines(bufnr, 0, -1, false)) do
-            if not empty_end and empty_start and line ~= "" then
-                empty_end = i - 1
-                break
+    if should_reorder(edits) then
```

Comment:
NOTE: I used stylua to format this file after adding this if. No changes were done to the original code other than the wrapping `if`.